client/components/Planning/CoveragePreview.jsx

```diff
@@ -0,0 +1,135 @@
+import classNames from 'classnames';
+import moment from 'moment-timezone';
+import {get} from 'lodash';
+import {getCoverageIcon, getCreator, getItemInArrayById, getDateTimeString, gettext} from '../../utils';
+import {UserAvatar, StateLabel} from '../index';
+
+export const CoveragePreview = ({coverage, users, desks, newsCoverageStatus, dateFormat, timeFormat, formProfile}) => {
+    const userAssigned = getCreator(coverage, 'assigned_to.user', users);
+    const deskAssigned = desks.find((d) =>
+        d._id === get(coverage, 'assigned_to.desk'));
+    const coverageDate = get(coverage, 'planning.scheduled');
+    const classes = classNames(
+        getCoverageIcon(get(coverage, 'planning.g2_content_type')),
+        {
+            'icon--green': coverageDate && moment(coverageDate).isAfter(moment()),
+            'icon--red': coverageDate && !moment(coverageDate).isAfter(moment()),
+        }
+    );
+    const coverageProvider = get(coverage, 'assigned_to.coverage_provider');
+    /* eslint-disable camelcase */
+    const {
+        assignor_user,
+        assignor_desk,
+        assigned_date_user,
+        assigned_date_desk,
+    } = coverage.assigned_to;
+
+    const deskAssignor = getItemInArrayById(users, assignor_desk);
+    const userAssignor = getItemInArrayById(users, assignor_user);
+    const coverageStatus = newsCoverageStatus.find((s) => s.qcode === coverage.news_coverage_status.qcode);
+    const coverageDateText = !coverageDate ? 'Not scheduled yet' :
+        getDateTimeString(coverageDate, dateFormat, timeFormat);
+    const assignmentPriority = get(coverage, 'assigned_to.priority');
+    const keywordText = get(coverage, 'planning.keyword.length', 0) === 0 ? '' :
+        coverage.planning.keyword.join(', ');
+
+    const coverageListItem = (
+        <Item noBg={true}>
+            <Column border={false}>
+                <Row>
+                    <i className={classes}/>&nbsp;&nbsp;
+                    {userAssigned && <UserAvatar user={userAssigned}/>}
+                    {!userAssigned && !deskAssigned && <span>Not assigned</span>}
+                    {deskAssigned && <span className="sd-overflow-ellipsis sd-list-item--element-grow">
+                        {'Desk:' + get(deskAssigned, 'name')}
+                    </span>}
+                    <i className="icon-time"/>
+                    <time><span>{coverageDateText}</span></time>
+                </Row>
+            </Column>
+        </Item>
+    );
+
+    let coverageTopBar = (<PreviewRow><label>Unassigned</label></PreviewRow>);
+
+    if (deskAssigned || userAssigned) {
+        coverageTopBar = (
+            <div>
+                <div className="TimeAndAuthor">
+                    { deskAssigned && <div>
+                        Desk:&nbsp;
+                        <span className="TimeAndAuthor__author">{deskAssigned.name.toUpperCase()}</span>
+                        {' (' + moment(assigned_date_desk).format('HH:mm DD/MM/YYYY') + ', ' +
```

Comment:
Can you please remove the hardcoded dateformat? I understand that it was done based on the initial request. Let's have the date format using config and time part after date (`DD/MM/YYYY HH:mm`).